### PR TITLE
[lang/ru] Update Russian language

### PR DESCRIPTION
**Pull request by mrmlnc**
The current state of the development branch.

### CODE REVIEW

**Comment by riuson** [comment]
What reason for replace " to ' and vice versa?

**Comment by mrmlnc** [comment]
Formatting corresponds to the original file, for example, [Backend/en/lang.php](https://github.com/octobercms/october/blob/master/modules/backend/lang/en/lang.php).

File: modules/cms/lang/ru/lang.php [hunk 1]
```diff
@@ -2,32 +2,80 @@
...
-        'file_already_exists' => 'Файл ":name" уже существует.',
-        'error_saving' => 'Ошибка сохранения файла ":name". Пожалуйста, проверьте права на запись.',
+        'invalid_file' => 'Ошибка в имени файла: :name. Имена файлов могут содержать только латинские буквы, цифры, знаки подчеркивания и точки. Пример правильных имен файлов: page.htm, page, subdirectory/page',
+        'invalid_property' => "Параметр ':name' нельзя изменить.",
+        'file_already_exists' => "Файл ':name' уже существует.",
+        'error_saving' => "Ошибка сохранения файла ':name'. Пожалуйста, проверьте права на запись.",
         'error_creating_directory' => 'Ошибка создания директории :name. Пожалуйста, проверьте права на запись.',
         'invalid_file_extension'=>'Указано неправильное расширение файла: :invalid. Разрешенные расширения: :allowed.',
-        'error_deleting' => 'Невозможно удалить файл шаблона :name. Пожалуйста, проверьте права на запись.',
+        'error_deleting' => "Невозможно удалить файл шаблона '':name'. Пожалуйста, проверьте права на запись.",
```
Comment (riuson) [inline modules/cms/lang/ru/lang.php]: Typo ' ' : name'

Comment (mrmlnc) [inline modules/cms/lang/ru/lang.php]: Oh, this Atom. Thank you!

**Comment by riuson** [comment]
Thanks.
As I see it, mainly used an apostrophe, if the string itself does not contain it. If the string contains an apostrophe, then used double quotes.
Only strings in 'relation' => [....]   do not meet this possible rule. But this was changed twice or more, in https://github.com/octobercms/october/commit/c43680b2ef28885a4511f8dce0fa2c3da415f9bf#diff-73ae1299d2c352fefb63eb86c178f48d and https://github.com/octobercms/october/commit/e0f1f53b2be1924b30d5fb0be8a7edc40a80e814#diff-73ae1299d2c352fefb63eb86c178f48d

**Comment by daftspunk** [comment]
Thanks!

**Pull request merged**